pod-security/baseline/disallow-host-ports/disallow-host-ports.yaml

```diff
@@ -26,7 +26,7 @@ spec:
           spec:
             =(initContainers):
               - =(ports):
-                  - X(hostPort): 0
+                  - X(hostPort): "null"
```

Comment:
Why does this value need to be "null"?

Reply:
Added Validation for negation anchor value must be null

Reply:
I think you might have been confused by that issue. The value is irrelevant in Kyverno's eyes because it only looks at the key. The value of a field using the negation anchor on the key could just as well be undefined. The relevant issue is only for validating the policy itself.